packages/atomic/cypress/integration/facets/date-facet/date-facet.cypress.ts

```diff
@@ -81,69 +82,59 @@ describe('Date Facet Test Suites', () => {
           0
         );
       });
-    });
 
-    describe('When user deselects 1 date-facet checkbox', () => {
-      function setupDateFacetWithCheckboxDeSelected() {
-        setupDateFacet();
-        cy.wait(RouteAlias.search);
-        cy.wait(RouteAlias.analytics);
-        selectFacetValueAt(0, dateField, dateFacetComponent);
-        cy.wait(RouteAlias.search);
-        cy.wait(RouteAlias.analytics);
-        selectFacetValueAt(0, dateField, dateFacetComponent);
-        cy.wait(RouteAlias.search);
-      }
+      describe('When user deselects 1 date-facet checkbox', () => {
+        function setupDateFacetWithCheckboxDeSelected() {
+          setupDateFacetWithCheckboxSelected();
+          cy.wait(RouteAlias.analytics);
+          selectFacetValueAt(0, dateField, dateFacetComponent);
+          cy.wait(RouteAlias.search);
+        }
 
-      describe('verify rendering', () => {
-        before(setupDateFacetWithCheckboxDeSelected);
-        FacetAssertions.assertCheckboxDisplay(
-          0,
-          false,
-          dateField,
-          dateFacetComponent
-        );
-      });
+        describe('verify rendering', () => {
+          before(setupDateFacetWithCheckboxDeSelected);
+          FacetAssertions.assertCheckboxDisplay(
+            0,
+            false,
+            dateField,
+            dateFacetComponent
+          );
+        });
 
-      describe('verify analytics', () => {
-        beforeEach(setupDateFacetWithCheckboxDeSelected);
-        FacetAssertions.assertAnalyticLogFacetDeselect(dateField);
+        describe('verify analytics', () => {
+          beforeEach(setupDateFacetWithCheckboxDeSelected);
+          FacetAssertions.assertAnalyticLogFacetDeselect(dateField);
+        });
       });
-    });
 
-    describe('When user clicks ClearAll facet button', () => {
-      function setupFacetWithTwoCheckboxesSelected() {
-        setupDateFacet();
-        cy.wait(RouteAlias.analytics);
-        selectFacetValueAt(0, dateField, dateFacetComponent);
-        cy.wait(RouteAlias.search);
-        cy.wait(RouteAlias.analytics);
-        selectFacetValueAt(1, dateField, dateFacetComponent);
-        cy.wait(RouteAlias.search);
-        cy.wait(RouteAlias.analytics);
-        selectClearAllFacetsButton(dateField, dateFacetComponent);
-        cy.wait(RouteAlias.search);
-      }
+      describe('When user clicks ClearAll facet button', () => {
+        function setupFacetWithTwoCheckboxesSelected() {
+          setupDateFacetWithCheckboxSelected();
```

Comment:
This function name gave me the impression that only one Checkbox is selected.

Reply:
That's the case, we select one value on line 55

Reply:
Then maybe, I am misunderstanding something. For this test do we not want two checkboxes selected?

Reply:
Ahh the method name is setupFacetWithTwoCheckboxesSelected... yes

Perhaps we could select two, but I think one should be enough